### ISSUE TITLE
Having Trouble with an Invisible Captcha on Form Submission – Any Solutions?

### ISSUE THREAD

**Issue by JakovGlavac** [QIN2DIM/hcaptcha-challenger#763]
Hello,
I'm attempting to submit a form with an invisible captcha, and even after successfully completing the challenge, I'm still receiving an error. I've tried various solutions, but I can't seem to determine why this is happening. Any suggestions?

![image](https://github.com/QIN2DIM/hcaptcha-challenger/assets/34971091/512b5c16-5805-4357-8a81-c2d67906b459)

I'm not using hcaptcha-challenger, but even when solving it by hand form submition fails. Why could it be?

**Comment by QIN2DIM** [comment]
IP?

**Comment by JakovGlavac** [comment]
Yeah but if I use regular chrome that is not controled everything works. That's why I'm confused.

**Comment by QIN2DIM** [comment]
Can you tell me what site you're testing? It's quite complicated, to be honest. A lot of times test sites block your workflow not because "you're a robot," but because your chain of behavior just happens to trigger a preset IF rule. (

**Comment by QIN2DIM** [comment]
Whether changing the agent IP address, switching from browser driver/agent to regular browser, or using undetected-selenium (patched chrome), The core goal is to get around these IF rules that are known only to the developer, and these methods usually work, but there are some special cases where they don't. I've even seen some sites where they will block your IP when you F12 view developer tools.🤦‍♂️

**Comment by JakovGlavac** [comment]
So here is the link: https://jobs.lever.co/spotify/ff3ea4a1-c7f4-44c8-8a34-70a1b1eb1e92/apply. I've observed that when a request is unsuccessful, they transmit all the form data to the server using a multipart form, but when it's successful, it only sends a cookie with an ID. However, the data is transmitted through hCaptcha. 

Is the undetected-selenium better than playwright_stealth or playwright_undetected?

**Comment by QIN2DIM** [comment]
playwright is better for me.
